src/udev/udev-varlink.c

```diff
@@ -188,6 +203,7 @@ int manager_open_varlink(Manager *m) {
                         "io.systemd.service.Reload", vl_method_reload,
                         "io.systemd.service.SetLogLevel", vl_method_set_log_level,
 
+                        "io.systemd.udev.Exit", vl_method_exit,
```

Comment:
still think this should be generic, i.e. io.systemd.service.Exit()

Reply:
I guess I've missed this one... But is it even needed? Wouldn't it be better to deprecate `udevadm control --exit`? It doesn't work as expected anyway in systemd world (i.e., it effectively causes _restart_ of `systemd-udevd` instead of _stop_; the man page even warns about this).

Reply:
yeah, sounds ok to kill